src/core/public/application/ui/app_container.tsx

```diff
@@ -126,9 +127,15 @@ export const AppContainer: FunctionComponent<Props> = ({
     setIsMounting,
   ]);
 
+const history = useHistory();
+
+const goToHomePage = () => {
```

Comment:
Why did you pull this function to the higher component? Cant this function can be inside the `app_not _found_screen.tsx`? Also use the path `/`, the router inside the app should in theory take care of rerouting this to the appropriate homepage.